### PR TITLE
style/oui-navbar

### PR DESCRIPTION
**Pull request by AxelPeter**
UU-583
UU-584
UU-585
UU-588

Staging doc: http://dev_style_oui_navbar.ui-kit.ovh/#!/ovh-ui-kit/navbar
Staging navbar: http://build_demo_navbar.ui-kit.ovh/#!/documentation/introduction

Linked to https://github.com/ovh-ux/ovh-documentation-toolkit/pull/15

### CODE REVIEW

**Comment by jimmyfortinx** [comment]
(optional) Outline needs to be fixed, but later since UI specs are not ready, so it is not mandatory.

File: packages/oui-navbar/_mixins.less [hunk 1]
```diff
@@ -120,65 +128,59 @@
...
   ) {
-    width: @width;
     font-size: @font-size;
+    font-weight: @font-weight;
     color: @font-color;
     background-color: @background-color;
 
     &:hover,
     &:focus {
+      // color: @font-color_hover;
```
Comment (neolitec) [inline packages/oui-navbar/_mixins.less]: A comment to remove

Comment (AxelPeter) [reply packages/oui-navbar/_mixins.less]: done

File: packages/oui-navbar/README.md [hunk 2]
```diff
@@ -149,9 +202,10 @@ Define the base styles of a backdrop for a navbar.
 
 ```less
 #oui > .navbar-backdrop(
```
Comment (tremblaymath) [inline packages/oui-navbar/README.md]: There is no example with navbar-backdrop

Comment (AxelPeter) [reply packages/oui-navbar/README.md]: This need javascript to toggle value of `aria-expanded` to be visible. 
Useless if put in the examples without context.

We'll need a single page with a full example for this (like for an example for fixed menu).

**Pull request merged**